qx/handle_difficulty.go

```diff
@@ -31,7 +32,7 @@ func Uint64ToCompact(input string) {
 	fmt.Printf("%d\n", diffCompact)
 }
 
-func CompactToGPS(diff string,edgeBits ,blockTime ,height int) {
+func CompactToGPS(diff string,edgeBits ,blockTime ,height int,p *params.Params) {
```

Comment:
For the same reason, the `qx` is just a utility, should not aware of consensus such as 45 days constrict.